tsdb/shard.go

```diff
@@ -524,16 +524,6 @@ func (s *Shard) SeriesN() int64 {
 	return s.engine.SeriesN()
 }
 
-// SeriesSketches returns the series sketches for the shard.
-func (s *Shard) SeriesSketches() (estimator.Sketch, estimator.Sketch, error) {
```

Comment:
Why is this removed?

Reply:
Do we still need series sketches if we aren't merging multiple files? It seems like we'd be able to have an accurate count at all times with a single series file.